IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TrackFacilityPart/DocPropertySet.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_TrackFacilityPart" UniqueId="011923c1-fbf2-84f9-6215-e3ea43caebc9" ApplicableType="IfcFacilityPart" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Restrict the Applicable type please to the relevent PDTs of IfcRailwayPartTypeEnum

Reply:
Revised https://github.com/bSI-InfraRoom/IFC-Specification/pull/164/commits/ab85f083c2717d30ef42c6a47dcce352a5b16e89